### DISCUSSION TITLE
looks pandas==1.4.2 can't install success with github actions workflow

### DISCUSSION THREAD

**Discussion by baiyyee** [community/community#33246] [Actions]
I have tried to install specific pandas version like 1.4.2 with github actions workflow, but it looks can't find this version, use `pip install pandas==1.4.2 -i https://pypi.python.org/simple` didn't not any help, too. what's wrong here?

workflow config like:
```
      - name: Install dependencies
        run: |
          pip install pandas==1.4.2 -i https://pypi.python.org/simple
          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Error as below:
![image](https://user-images.githubusercontent.com/4414171/190608325-cf07016c-0653-4981-b395-a40cd1f60bef.png)

**Comment by airtower-luna** [comment]
According to the error message Pandas 1.4 requires Python 3.8, but your workflow is using Python 3.7. Make sure to use an up to date Python version, using [`actions/setup-python`](https://github.com/actions/setup-python) is usually the right approach. :slightly_smiling_face: :snake:
